libyul/optimiser/SSAValueTracker.cpp

```diff
@@ -33,6 +33,16 @@ void SSAValueTracker::operator()(Assignment const& _assignment)
 		m_values.erase(var.name);
 }
 
+set<YulString> SSAValueTracker::ssaVariables(Block const& _ast)
```

Comment:
If I was overly picky, I'd say this is a weird place in the file to put it and it should rather go after the ``operator()``s...